### ISSUE TITLE
Errors when running "How to use Tax-Calculator"

### ISSUE THREAD

**Issue by GoFroggyRun** [open-source-economics/Tax-Calculator#1721]
I might be missing something, but when I run the "How to use Tax-Calculator" file (aka "10_Minutes_to_Tax-Calculator"), I encounter the following errors:

![screen shot 2017-12-04 at 3 03 07 pm](https://user-images.githubusercontent.com/13324931/33573620-54564bb6-d904-11e7-9ee5-df4334c8e3d4.png)

![screen shot 2017-12-04 at 3 03 12 pm](https://user-images.githubusercontent.com/13324931/33573618-53505cac-d904-11e7-8d5e-6dedc80c7dee.png)

I' m currently on master branch, in env taxcalc-dev, and all tests pass:
```
(taxcalc-dev) Sean-Wangs-MBP:taxcalc seanwang$ py.test
==================================== test session starts ====================================
platform darwin -- Python 2.7.14, pytest-3.3.0, py-1.5.2, pluggy-0.6.0
rootdir: /Users/seanwang/Documents/GIT/tax-calculator, inifile: setup.cfg
plugins: xdist-1.20.1, forked-0.2, hypothesis-3.38.5, pep8-1.0.6
collected 385 items                                                                         

tests/test_4package.py ..                                                             [  0%]
tests/test_behavior.py .......                                                        [  2%]
tests/test_calculate.py ..........................                                    [  9%]
tests/test_consumption.py .......                                                     [ 10%]
tests/test_cpscsv.py ..                                                               [ 11%]
tests/test_decorators.py ................                                             [ 15%]
tests/test_docs.py .                                                                  [ 15%]
tests/test_functions.py ..                                                            [ 16%]
tests/test_growdiff.py .......                                                        [ 18%]
tests/test_growfactors.py ...                                                         [ 18%]
tests/test_macro_elasticity.py .                                                      [ 19%]
tests/test_notebooks.py .                                                             [ 19%]
tests/test_parameters.py ...................                                          [ 24%]
tests/test_policy.py ...............................                                  [ 32%]
tests/test_puf_var_stats.py .                                                         [ 32%]
tests/test_pufcsv.py .....                                                            [ 34%]
tests/test_records.py ............                                                    [ 37%]
tests/test_reforms.py ............................................................... [ 53%]
                                                                                      [ 53%]
tests/test_responses.py .                                                             [ 53%]
tests/test_taxcalcio.py .....................                                         [ 59%]
tests/test_tbi.py ................                                                    [ 63%]
tests/test_utils.py ..................................                                [ 72%]
tests/filings/forms/test_tax_form.py ..............                                   [ 75%]
tests/filings/forms/test_tax_forms.py ............................................... [ 88%]
...........................                                                           [ 95%]
validation/taxsim/test_simpletaxio.py ...................                             [100%]

=============================== 385 passed in 578.47 seconds ================================
```

Can anyone help?
@MattHJensen @martinholmer @hdoupe

**Comment by martinholmer** [comment]
@GoFroggyRun,  I don't know how to use notebooks, so that may be a problem in what I say below.

I can't replicate the problem you report in issue #1721.  When I'm on the master branch in the top Tax-Calculator directory with `puf.csv` there along with a script that includes the Python statements from the notebook, the command `python notebook.py` works fine.  Here is what I get:
```
iMac2:tax-calculator mrh$ python notebook.py
You loaded data for 2009.
Tax-Calculator startup automatically extrapolated your data to 2013.
You loaded data for 2009.
Tax-Calculator startup automatically extrapolated your data to 2013.
-103216230236.0
```
Notice the result is slightly different from what's in the notebook, probably because the notebook hasn't been updated since the new puf-related files were introduced a month ago in pull request #1624.

But I'm not getting the errors you report; the script runs just fine.  Here is the `notebook.py` script:
```
iMac2:tax-calculator mrh$ cat notebook.py
from taxcalc import *

# The baseline includes AMT repeal. 
p_xx = Policy()
reform_xx = {
    2017: {
        '_AMT_rt1': [0.0],
        '_AMT_rt2': [0.0]
    }
}
p_xx.implement_reform(reform_xx)

# The reform expands the second tax bracket
# (and repeals AMT to match the baseline)

p_yy = Policy()
reform_yy = {
    2017: {
        '_II_brk2': [[65005, 130010, 65005, 88180, 0]],
        '_AMT_rt1': [0.0],
        '_AMT_rt2': [0.0]
    }
}
p_yy.implement_reform(reform_yy)

c_xx = Calculator(policy=p_xx, records=Records("puf.csv"))
c_yy = Calculator(policy=p_yy, records=Records("puf.csv"))

c_xx.advance_to_year(2017)
c_yy.advance_to_year(2017)
assert c_xx.current_year == 2017
assert c_yy.current_year == 2017
c_xx.calc_all()
c_yy.calc_all()

d = ((c_yy.records.combined - c_xx.records.combined)*c_xx.records.s006).sum()
print d
```

Can you tell us exactly how you are starting the notebook?
If you have discovered a bug, you need to explain how others can replicate the buggy behavior.

**Comment by GoFroggyRun** [comment]
@martinholmer thanks for your attention on this issue. 

I figured out the cause of the problem, which has something to do with my conda environment. When I was on developer environment, `taxcalc-dev`, where `taxcalc` package is NOT installed, the notebook should load `taxcalc` package on my root environment. The version of which is out-of-date, and thus causing the errors above. I have no idea why this is the case.

By removing `taxcalc` package on root environment, I no longer have problems with `taxcalc-dev` package nor notebook. Closing.
